nozzle.go

```diff
@@ -18,7 +18,7 @@ import (
 	"log"
 	"time"
 
-	"github.com/cloudfoundry/noaa"
+	noaa_consumer "github.com/cloudfoundry/noaa/consumer"
```

Comment:
Consumer package is imported with name `noaaConsumer` [here](https://github.com/rakutentech/go-nozzle/blob/2cf5fdab5e9ae02d39a8daa43af564667a81f885/consumer.go#L9) , let's keep the name same.